lightning: add load data doc (#12959)
pingcap/docs: error-codes.md

* Error Number: 8154

    Currently `LOAD DATA` does not support importing data locally from TiDB server. You can specify `LOCAL` to import from client, or upload data to S3 and then import it. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md).

* Error Number: 8155

    Currently, `LOAD DATA` only supports importing Parquet files from S3. You can upload Parquet files to S3 and then import it. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md).

* Error Number: 8156

    The file path of the `LOAD DATA` statement cannot be empty. You need to set the correct path before importing. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md).

* Error Number: 8157

    Unsupported data formats. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) to check the supported data formats.

* Error Number: 8158

>     The S3 path passed in is invalid. See [external storage](/br/backup-and-restore-storages.md) to set a valid path.

Reviewer: ```suggestion
    The S3 path entered in is invalid. See [external storage](/br/backup-and-restore-storages.md) to set a valid path.
```

* Error Number: 8159

>     TiDB cannot access the S3 path passed in the `LOAD DATA` statement. Make sure that the S3 bucket you entered exists, the TiDB server has access to S3, and you have privileges to access the corresponding bucket.

Reviewer: ```suggestion
    TiDB cannot access the S3 path entered in the `LOAD DATA` statement. Make sure that the S3 bucket you entered exists, or you have entered the correct access key and secret access key to let TiDB access the bucket.
```

* Error Number: 8160

    `LOAD DATA` fails to read the data file. Refer to the specific error message for action.

* Error Number: 8162

>     There is an error in the `LOAD DATA` statement. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) for supported functions.

Reviewer 2: ```suggestion
    There is an error in the `LOAD DATA` statement. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) for supported options.
```

* Error Number: 8163

    Unknown `LOAD DATA ... WITH ... ` option. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) for supported options.

* Error Number: 8164

    The `LOAD DATA ... WITH ... ` option takes an invalid value. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) for valid values.

* Error Number: 8165

>     Repeatedly specify the `LOAD DATA ... WITH ... ` option. Each option can be specified only once.

Reviewer 2: ```suggestion
    Specify duplicated `LOAD DATA ... WITH ... ` option. Each option can be specified only once.
```

* Error Number: 8166

    Some `LOAD DATA ... WITH ... ` options can only be used in certain import modes. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) to see the supported options.

* Error Number: 8170

    The specified `LOAD DATA` job does not exist or is not created by the current user. Currently you can only view jobs that you created.

* Error Number: 8171

>     Perform operation and maintenance for unsupported `LOAD DATA` task status. Refer to the specific instructions for processing.

Author: ```suggestion
    You cannot perform operation and maintenance for unsupported `LOAD DATA` task status. Refer to the specific instructions for processing.
```

* Error Number: 8172

>     `LOAD DATA` specified with `LOCAL` cannot be run in the background. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) to change the SQL statement.

Reviewer 3: ```suggestion
    `LOAD DATA` specified with `LOCAL` cannot be run in the background. Only `LOAD DATA` specified with S3 or GCS path can be run in the background. See [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) to change the SQL statement.
```
